code/__defines/_planes+layers.dm

```diff
@@ -191,21 +191,23 @@ What is the naming convention for planes or layers?
 	#define BLIND_LAYER         3
 	#define CRIT_LAYER          4
 
-#define VISION_CONE_PLANE               7
+#define VISION_CONE_PLANE				7
```

Comment:
dude, spaces are there for a reason, either bring them back or remove alltogether